### ISSUE TITLE
Trouble running chimpanzee example

### ISSUE THREAD

**Issue by PeetoomHeida** [StanJulia/Stan.jl#109]
I think the chimpanzees.jl example (in Stan.jl/Examples/Basic_intros/) is missing a dependency for DataFrames.jl, and possibly AxisKeys.jl. I ran into an error when trying to run the code that "DataFrame" was not defined. Adding DataFrames to the "using" line resolved. Got the same error with the axiskeys command on line 47. After importing AxisKeys, I now get the following issue:
```
LoadError: MethodError: no method matching axiskeys(::StanTable{Matrix{Float64}})
```
Any idea what is going on here?

**Comment by goedman** [comment]
Thank you for the report, will have a look and get back to you.

**Comment by goedman** [comment]
Found the problem but also noticed an issue in StanSample.jl which will take a bit more work.

Above error is due to the fact that at some point I made `:keyedarray` the default output_format in read_samples(). When I switched this back to `:table` I did not update all example accordingly. If you use:
```
chns = read_samples(model, :keyedarray)
```
axiskeys(cons) should work again.

But after fixing StanSample I'll redo the Basic_intros (and in fact split them in separate directories).

Rob
